examples/acquiring_data/fido_metadata_queries.py

```diff
@@ -0,0 +1,53 @@
+"""
+====================================
+Querying Metadata clients using Fido
+====================================
+
+This example shows how to search and retrieve metadata using `~sunpy.net.Fido` from
+search facilities like `~sunpy.net.hek.HEKClient`, `~sunpy.net.helio.HECClient`,
+and `~sunpy.net.jsoc.JSOCClient`. It also shows how to display desired columns from the result.
+"""
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###################################################################
+# We will query Helio for the 'rhessi_flare_list' table and
+# limit the total number of records to 5.
+# For the same time range, we will query HEK for 'FL' as the
+# event type and 'PeakFlux' greater than 1000.
+# We will also search JSOC for 'hmi.m_45s' series.
+timerange = a.Time('2010/8/1 03:40', '2010/8/1 3:40:10')
+results = Fido.search(timerange, a.helio.TableName('rhessi_hxr_flare') |
+                      a.hek.FL & (a.hek.FL.PeakFlux > 1000) |
+                      a.jsoc.Series('hmi.m_45s') & a.jsoc.Notify("jsoc@cadair.com"))
```

Comment:
I am not a great fan of using this email address in our examples, if people copy it it might cause problems for the JSOC folk. Do you think modifying this example to either use the VSO for fetch or not to fetch at all would substantially detract from it?

Reply:
Or we can do the email address hack in conf.py as we discussed in the community call.

Reply:
I have added it to docs/conf and access it via os.envirom.